models/classes/LtiProvider/LtiProviderService.php

```diff
@@ -93,6 +107,11 @@ private function aggregate($result, $method)
         return $result;
     }
 
+    /**
+     * @param string $id
+     *
+     * @return LtiProvider
+     */
```

Comment:
Should phpdoc be removed as it provides no useful information in addition to method signature?